### PR TITLE
Add chown

### PR DESCRIPTION
**Pull request by corradio**
This adds a new `chown` attribute to a configuration step.
It represents the user which will be used to copy input files.
This is required in cases where the base image uses a specific user. Docker by default copies files as `root`, and therefore we need to be able to change the user owning the files that will be copied.

### CODE REVIEW

File: brick/__main__.py [hunk 1]
```diff
@@ -101,19 +101,30 @@ def generate_dockerfile_contents(
...
     dockerfile_contents += f"FROM {from_image}\n"
+
+    copy_flag_chown = f" --chown={chown}" if chown else ""
+
     if inputs_from_build:
         dockerfile_contents += (
-            "\n".join([f"COPY --from={x[0]} /home/{x[1]} /home/{x[1]}" for x in inputs_from_build])
+            "\n".join(
+                [
+                    f"COPY{copy_flag_chown} --from={x[0]} /home/{x[1]} /home/{x[1]}"
```
Comment (skovhus) [inline brick/__main__.py]: Nitpick: but would be a bit easier to read this if we added an additional whitespace character here: `COPY {copy_flag_chown}`.

**Review by skovhus — COMMENTED** [review]
Pylint complaints about too many arguments: `[R0913(too-many-arguments), generate_dockerfile_contents] Too many arguments (11/10)`

File: brick/__main__.py (hunk 1 above)
Comment (corradio) [reply brick/__main__.py]: In this case the command becomes `COPY  --flag1 --flag2` (with an additional whitespace). Is this what you're suggesting? I didn't want to introduce a double whitespace in order to not mess up with the command line parser, but I'm happy to change it though!

**Comment by corradio** [comment]
> Pylint complaints about too many arguments: `[R0913(too-many-arguments), generate_dockerfile_contents] Too many arguments (11/10)`

Would it make sense to disable this line? What would you suggest?

File: brick/__main__.py (hunk 1 above)
Comment (skovhus) [reply brick/__main__.py]: Yes, I'm suggesting additional white space for readability. I'm fairly sure most command-line parses would handle multiple white spaces. 

We already have double white space it seems in https://github.com/tmrowco/brick/pull/77/files/8ff3dbc19b52ba67f589012351646cd5470832ea#diff-70115798985cf86a2ba2136857657c87b1888392550f3c723b2a9e02e87c2b95R122

**Comment by skovhus** [comment]
> Would it make sense to disable this line? What would you suggest?

10 is quite a lot of arguments, but the most pragmatic is to increase the number here https://github.com/tmrowco/brick/blob/master/pylintrc#L573

**Review by skovhus — APPROVED** [review]

**Pull request merged**